site/content/community/02-roadmap.md

```diff
@@ -1,9 +1,10 @@
 ---
 title: "Road Map"
 weight: 20
+url: community/road-map
 ---
 
-## Project Roadmap
+## Project Road Map
```

Comment:
It has to be one word. It's two words we talk about roads, but the project management term is a one word.

/cc @kalbir

Reply:
In any case, we don't have to address it in this PR :)

Reply:
https://dictionary.cambridge.org/dictionary/english/road-map